Makefile

```diff
@@ -32,46 +34,64 @@ clean:
 
 .PHONY: devkit
 devkit:
-	if $$(docker images | grep mesosphere/adminrouter-devkit | grep -q latest); then \
+	if $$(docker images | grep mesosphere/adminrouter-devkit | grep -q full); then \
 		echo "+ Devkit image already build"; \
 	else \
 		echo "+ Building devkit image"; \
-		docker rmi -f mesosphere/adminrouter-devkit:latest; \
 		docker build \
 			--rm --force-rm \
-			-t \
-			mesosphere/adminrouter-devkit:latest ./docker/ ;\
+			-t mesosphere/adminrouter-devkit:noresty \
+			-f ./docker/Dockerfile \
+				./docker/ && \
+		docker run \
```

Comment:
I guess this is a response to the https://github.com/dcos/adminrouter/pull/53/files#r102913232 comment.